src/ui/components/Icon/extensions-complex.svg

```diff
@@ -0,0 +1,165 @@
+
+<svg width="44px" height="48px" viewBox="0 0 44 48" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
As mentioned on irc - this new icon for the header isn't yet signed off so it might need a change before the tag. Phil is going to let us know before then.